support.md

```diff
@@ -1,286 +1,21 @@
-   to the APIs, such as disallowing access to the file system from rule implementations (because
-   it's not hermetic).
-3. We will enumerate the internal rule APIs, and make sure that they are appropriately marked (for
-   example with annotations) and documented. Just collecting a list will likely give us good
-   suggestions for further improvements, as well as opportunities for a more principled API review
-   process.
-4. We will work on removing (legacy) Google-internal features to reduce the amount of differences
-   between the internal and external rule sets.
-5. We will encourage our engineers to make changes in the external depot first, and migrate them to
-   to the internal one afterwards.
-6. We will move more of our rule implementations into the open source repository (Android, Go,
-   Python, the remaining C++ rules), even if we don't consider the code to be *ready* or if they are
-   still missing tools to work properly.
-
-
-### Stable
-We expect the following rules and features to be stable. They are widely used within Google, so
-our internal testing should ensure that there are no major breakages.
-
-<table class="table table-condensed table-striped table-bordered">
-  <colgroup>
-    <col class="support-col-rules" />
-    <col class="support-col-notes" />
-  </colgroup>
-  <thead>
-    <tr>
-      <th>Rules</th>
-      <th>Notes</th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td>C/C++ rules except <code>cc_toolchain</code> and <code>cc_toolchain_suite</code></td>
-      <td></td>
-    </tr>
-    <tr>
-      <td>Java rules except <code>java_toolchain</code></td>
-      <td></td>
-    </tr>
-    <tr>
-      <td>Android rules except <code>android_ndk_repository</code> and
-        <code>android_sdk_repository</code></td>
-      <td></td>
-    </tr>
-    <tr>
-      <td><code>genrule</code></td>
-      <td></td>
-    </tr>
-    <tr>
-      <td><code>genquery</code></td>
-      <td></td>
-    </tr>
-    <tr>
-      <td><code>test_suite</code></td>
-      <td></td>
-    </tr>
-    <tr>
-      <td><code>filegroup</code></td>
-      <td></td>
-    </tr>
-    <tr>
-      <td><code>config_setting</code></td>
-      <td>
-        <ul>
-          <li>This rule is used in <code>select()</code> expressions. We have hundreds of uses, so
-            we expect the basic functionality to be stable. That said, there are some common use
-            cases that are not covered yet, or that require workarounds. For example, it's not
-            easily possible to select on information specified in a CROSSTOOL file, such as the
-            target abi. Another example is that it's not possible to OR multiple conditions,
-            leading to duplicated entries in the select.
-          </li>
-        </ul>
-      </td>
-    </tr>
-  </tbody>
-</table>
-
-
-### Unstable
-These rules and features have known limitations that we will likely address in future releases.
+Before any major change, you will have advance notice on the
```

Comment:
Major releases are announced in advance on the

Reply:
Done.